src/benchmarklib/abstract_table_generator.cpp

```diff
@@ -275,4 +280,22 @@ AbstractTableGenerator::SortOrderByTable AbstractTableGenerator::_sort_order_by_
 void AbstractTableGenerator::_add_constraints(
     std::unordered_map<std::string, BenchmarkTableInfo>& table_info_by_name) const {}
 
+bool AbstractTableGenerator::_table_is_sorted_by(const std::shared_ptr<Table>& table,
+                                                 const SortColumnDefinition& sort_definition) {
```

Comment:
The way this looks to me is as if you are only checking if the individual chunks are sorted. That, however, does not mean that the entire table is sorted, too.

Reply:
Yes, this is an incorrect naming here. As it is already checked whether the table is sorted, we only make sure here that each chunk has the correct SortColumnDefinition as a consequence.